dusty/systems/docker/compose.py

```diff
@@ -3,11 +3,14 @@
 
 import yaml
 
-from . import _get_canonical_container_name, get_docker_env, get_docker_client, _get_dusty_containers
+from . import (_get_canonical_container_name, get_docker_env, get_docker_client,
+               _get_dusty_containers, _get_app_or_service_name_from_container,
+               _get_container_for_app_or_service)
```

Comment:
Going through and fixing all these underscores sounds like a task for another PR